.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       # configure AWS to run dev server necessary for `npm run test:ci`
       - uses: aws-actions/configure-aws-credentials@v4
         with:
-          aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+          aws-region: ${{ vars.AWS_DEFAULT_REGION }}
```

Comment:
On one hand, this reduces redundancy in having multiple `AWS_DEFAULT_REGION`s defined. On the other hand, this moves `AWS_DEFAULT_REGION` away from other related variables, some of which must be kept as secrets:

![image](https://github.com/nextstrain/nextstrain.org/assets/13424970/756c9168-da4e-4db5-90ae-73c213ce7e42)

Thoughts? Based on [code search](https://github.com/search?q=org%3Anextstrain+secrets.AWS_DEFAULT_REGION&type=code), this applies to nextstrain.org and forecasts-ncov.

Reply:
We've had to remove `AWS_DEFAULT_REGION` from secrets for any repo that uses the `pathogen-repo-build` workflow to prevent the AWS batch link from being borked (e.g. https://github.com/nextstrain/rsv/issues/53#issuecomment-1913717467). I think this move will match expectations that only "real" secrets should be kept as secrets and other shared values can be the unmasked variables.

Reply:
Thanks for the added context! I agree better to properly scope usage of the secrets store. I'll follow through with the move here and in forecasts-ncov.